src/main/scala/org/ergoplatform/nodeView/mempool/ExpiringApproximateCache.scala

```diff
@@ -22,26 +24,26 @@ sealed trait ApproximateCacheLike[T] {
   * Ie. we expire whole bloom filters instead of expiring elements and we check all bloom filters for element presence
   *
   * @param bloomFilterQueueSize how many bloom filters at maximum to keep in FIFO queue
-  * @param bloomFilterApproxElemCount approximate element size of a single bloom filter
   * @param bloomFilterQueue fifo collection of bloom filters with tracking index
   * @param frontCacheMaxSize maximum number of elements to keep in cache, following elems are kept in bloom filters
   * @param frontCacheElemExpirationMs for how long to keep elems in cache
   * @param frontCache time expiring cache in front of boom filters
   */
 case class ExpiringApproximateCache(
   bloomFilterQueueSize: Int,
-  bloomFilterApproxElemCount: Int,
   bloomFilterQueue: Vector[(Long, BloomFilter[String])],
   frontCacheMaxSize: Int,
   frontCacheElemExpirationMs: Long,
   frontCache: TreeMap[String, Long]
-) extends ApproximateCacheLike[String] {
+) extends ApproximateCacheLike[String] with ScorexLogging {
+
+  require(frontCacheMaxSize > 0)
 
-  private def createNewFilter =
+  private def createNewFilter: BloomFilter[String] =
     BloomFilter.create[String](
       Funnels.stringFunnel(Charset.forName("UTF-8")),
-      bloomFilterApproxElemCount,
-      0.05d
+      frontCacheMaxSize,
+      0.001d   // 0.1 % false positive rate
```

Comment:
It is better than previous value, but that is not a good comment

Reply:
Yes, I believe that since you change it then it is somehow better. But why it is better? Any comment in the code based on your experience would be helpful, while you still remember it :)

Reply:
idk why to write that bigger false positive rate is causing more false positives, seems trivial

Reply:
added a note on total FPR, which is not so trivial